src/app/external/animate-on-scroll/animate-on-scroll.directive.ts

```diff
@@ -0,0 +1,153 @@
+import { Directive, Input, Renderer2, ElementRef, OnInit, OnDestroy, AfterViewInit } from '@angular/core';
+import { ScrollService } from './scroll.service';
+import { Subscription } from 'rxjs';
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: '[animateOnScroll]',
+})
+export class AnimateOnScrollDirective implements OnInit, OnDestroy, AfterViewInit {
```

Comment:
This functionality isn't used yet.

Either implement this into the features component or remove it from the source.